### ISSUE TITLE
Color contrast of focus on learn more about ONNX Runtime&Generative AI, quick start, tutorials, Install ONNX Runtime, Hardware Accelartion,ONNX Runtime youtube channel is 1.152:1 less than 3:1: A11y_ONNX Runtime & Ecosystem_Runtime_Non text contrast

### ISSUE THREAD

**Issue by Govardhani-T** [microsoft/onnxruntime#24995]
#A11yTCS #A11yMAS #A11ySev2 #MAS1.4.11 #Non-text Contrast #DesktopWeb #Chrome #Win11 #BM_ONNX Runtime & Ecosystem_Web_June2025; #ONNX Runtime & Ecosystem;  #Benchmark #A11yWCAG2.1 #rev:moda
**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification."**
### Environment Details:
Application Name: WCP URLs - ONNX Runtime
URL: https://onnxruntime.ai/
Chrome Version: 120.0.6099.130
OS: Windows 11 version 23H2

### Steps to Reproduce

1. Hit the URL "https://onnxruntime.ai/".
2. "ONNX Runtime" home page appeared.
3. Tab till learn more about ONNX Runtime&Generative AI, quick start, tutorial, Install ONNX Runtime, Hardware Acceleration Links.
4. Observe that Color contrast of focus on learn more about ONNX Runtime&Generative AI, quick start, tutorials, Install ONNX Runtime, Hardware Accelartion,ONNX Runtime youtube channel is 1.152:1 less than 3:1
###Actual:
Color contrast of focus on learn more about ONNX Runtime&Generative AI, quick start, tutorials, Install ONNX Runtime, Hardware Accelartion,ONNX Runtime youtube channel links with the background color is 1.152:1 less than 3:1.
###Expected:
Color contrast of focus on learn more about ONNX Runtime&Generative AI, quick start, tutorials, Install ONNX Runtime, Hardware Accelartion,ONNX Runtime youtube channel links with background color should be equal or greater than 3:1
###User Impact:
Low vision users will face difficulties to know the focused item if the Color contrast of focus on learn more about ONNX Runtime&Generative AI, quick start, tutorials, Install ONNX Runtime, Hardware Accelartion,ONNX Runtime youtube channel is less than 3:1
###Recommendations:
Refer[ Developer Resources (sharepoint.com)](https://microsoft.sharepoint.com/sites/accessibility/SitePages/Developer-Resources.aspx).
### MAS reference:
[MAS 1.4.11 – Non-text Contrast](https://aka.ms/MAS1.4.11)
Check out [MAS 1.4.11: Non-Text Contrast Guide](https://microsoft.sharepoint.com/sites/accessibility/SitePages/MAS-1.4.11-Non-Text-Contrast-Guide.aspx?
### Attachment:
<img width="877" alt="Image" src="https://github.com/user-attachments/assets/2fc1d8b1-1a4c-4b77-90dd-05e4da058a05" />
